src/applications/mhv-medical-records/util/dateHelpers.js

```diff
@@ -128,11 +128,33 @@ export const currentDateAddHours = hours => {
   return formatUtcTime(newDate);
 };
 
+/**
+ * Current date minus 1 minute and plus 1 hour formatted "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"
+ * @example 2025-11-20T19:14:57.282Z
+ * @returns {String}
+ */
+export const currentDateAddOneHourMinusOneMinute = () => {
+  const newDate = subMinutes(addHours(new Date(), 1), 1);
+  return formatUtcTime(newDate);
+};
```

Comment:
Do we need this function? Is it used anywhere?

Reply:
Finished removing